api/db/database-builder/database-builder.js

```diff
@@ -160,7 +160,9 @@ class DatabaseBuilder {
       from pg_class ref
       join pg_namespace rs on rs.oid = ref.relnamespace
       join pg_constraint c on c.contype = 'f' and c.conrelid = ref.oid
-      join fk_tree p on p.reloid = c.confrelid ), all_tables as (
+      join fk_tree p on p.reloid = c.confrelid
+      where ref.oid != p.reloid),
```

Comment:
Dans le commentaire du commit sur `api/db/database-builder/database-builder.js` il me semble qu'il serait pertinent de remettre un lien sur https://stackoverflow.com/questions/51279588/sort-tables-in-order-of-dependency-postgres indiquant qu'on y trouve le code `where ref.oid != p.reloid  -- do not enter to tables referencing theirselves.` et que cela nous conforte dans l'idée d'ajouter ce code.